CMakeLists.txt

```diff
@@ -38,6 +38,16 @@ option(CPPKAFKA_BOOST_STATIC_LIBS "Link with Boost static libraries." ON)
 option(CPPKAFKA_BOOST_USE_MULTITHREADED "Use Boost multithreaded libraries." ON)
 option(CPPKAFKA_RDKAFKA_STATIC_LIB "Link with Rdkafka static library." OFF)
 
+# Determine if this is a 32 or 64 bit build
+string(FIND ${CMAKE_CXX_FLAGS} "-m64" BITNESS)
```

Comment:
Is there any guarantees there's actually going to be a `-m64` in CXX_FLAGS? I've used something like [this](https://dilawarnotes.wordpress.com/2016/01/13/detect-architecture-of-your-system-using-cmake/) in the past to detect 32/64 bit architectures.

Reply:
Yes I think your proposal is more platform independent. How about something like this which is simpler? from [the 2nd voted answer](https://stackoverflow.com/questions/4029587/compiling-32-bit-vs-64-bit-project-using-cmake)
```
math(EXPR BITS "8*${CMAKE_SIZEOF_VOID_P}")
set(BOOST_LIBRARY "/boost/win${BITS}/lib")
```

Reply:
Yeah, that would also work